docs/articles/documentation/test-api/actions/action-options.md

```diff
@@ -10,6 +10,7 @@ This topic describes [test action](README.md) options.
 
 * [Basic Action Options](#basic-action-options)
 * [Mouse Action Options](#mouse-action-options)
+* [Drag Action Options](#drag-action-options)
```

Comment:
It should be `DragToElementOptions` (like in the TypeScript definitions). Otherwise it would be strange it the `Drag` action doesn't use `Drag Action Options`